### PR TITLE
Button action event added

### PR DESCRIPTION
**Pull request by apsavin**
What about unified action event for buttons?

### CODE REVIEW

File: desktop.blocks/button/button.js [hunk 1]
```diff
@@ -80,8 +80,13 @@ BEM.DOM.decl('button', /** @lends Button.prototype */ {
 
         'pressed' : function(modName, modVal) {
 
-            this.isDisabled() || this.trigger(modVal == 'yes' ? 'press' : 'release');
```
Comment (tadatuta) [inline desktop.blocks/button/button.js:83]: we should check if the button isn't disabled

File: desktop.blocks/button/button.js [hunk 2]
```diff
@@ -80,8 +80,13 @@ BEM.DOM.decl('button', /** @lends Button.prototype */ {
...
-
+            if (modVal == 'yes') {
+		this.trigger('press');
```
Comment (tadatuta) [inline desktop.blocks/button/button.js:84]: indentation

**Comment by dfilatov** [comment]
Can you provide a real case where unified actions is useful?

**Comment by apsavin** [comment]
Well, I think, we can just use 'release' button event.

**Pull request closed without merging**